administration/src/components/verification/VerificationController.tsx

```diff
@@ -0,0 +1,124 @@
+import { useAppToaster } from '../AppToaster'
+import { CARD_PADDING } from '../applications/ApplicationsOverview'
+
+const ApplicationViewCard = styled(Card)`
+  margin: 10px;
+`
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  justify-content: center;
+  padding: ${CARD_PADDING}px;
+`
+
+const CenteredMessage = styled(NonIdealState)`
+  margin: auto;
+`
+
+const ButtonContainer = styled.div`
+  display: flex;
+  width: inherit;
+  flex-direction: row;
+  justify-content: space-around;
+`
+
+type ApplicationVerificationProps = {
+  applicationVerificationAccessKey: string
+}
+
+type VerifiedState = 'verified' | 'rejected' | 'pending'
+
+const ApplicationVerification = ({ applicationVerificationAccessKey }: ApplicationVerificationProps) => {
+  const [verified, setVerifiedState] = useState<VerifiedState>('pending')
+  const config = useContext(ProjectConfigContext)
+  const appToaster = useAppToaster()
+  const showErrorToaster = () => appToaster?.show({ intent: 'danger', message: 'Etwas ist schief gelaufen.' })
+  const [verifyOrRejectApplicationVerification, { loading: verificationLoading }] =
+    useVerifyOrRejectApplicationVerificationMutation({
+      onError: error => {
+        console.error(error)
+        showErrorToaster()
+      },
+      onCompleted: ({ result }, clientOptions) => {
+        if (!result) {
+          console.error('Verify operation returned false.')
+          showErrorToaster()
+        } else {
+          if (!clientOptions?.variables?.verified) setVerifiedState('rejected')
+          else setVerifiedState('verified')
+        }
+      },
+    })
+
+  const submitApplicationVerification = (verified: boolean) => {
+    verifyOrRejectApplicationVerification({
+      variables: {
+        accessKey: applicationVerificationAccessKey,
+        verified,
+      },
+    })
+  }
+
+  const { loading, error, data, refetch } = useGetApplicationByApplicationVerificationAccessKeyQuery({
+    variables: { applicationVerificationAccessKey },
+  })
+
+  if (loading) return <Spinner />
+  else if (error || !data) return <ErrorHandler refetch={refetch} />
+  if (data.verification.rejectedDate || data.verification.verifiedDate)
+    return <CenteredMessage title='Dieser Antrag wurde bereits bearbeitet' />
+  if (verified !== 'pending')
+    return (
+      <CenteredMessage
+        title={`Der Antrag wurde Erfolgreich ${verified === 'verified' ? 'verifiziert' : 'abgelehnt'}`}
+        description='Sie können das Fenster jetzt schließen.'
+      />
+    )
+
+  const { jsonValue, id } = data.application
+  const jsonField = JSON.parse(jsonValue)
+  const baseUrl = `${process.env.REACT_APP_API_BASE_URL}/application/${config.projectId}/${id}`
+  return (
+    <Container>
+      <H1>{config.name}</H1>
+      <span>
+        Können Sie die Angaben, welche die Organisation <b>{data.verification.organizationName}</b> betreffen
+        bestätigen?
```

Comment:
I think some more context is missing:
Proposal:

Guten Tag {data.verification.contactName},

Sie wurden gebeten, die Angaben eines Antrags auf Ehrenamtskarte zu bestätigen.
Die Antragsstellerin oder der Antragssteller hat Sie als Kontaktperson der Organisation {data.verification.organizationName} angegeben.
Im Folgenden können Sie den zugehörigen Antrag einsehen.
Wir bitten Sie, die enthaltenen Angaben, welche die Organisation {data.verification.organizationName} betreffen, zu bestätigen. Falls Sie denken, die Angaben wurden fälschlicherweise gemacht, bitten wir Sie, den Angaben zu widersprechen.

Reply:
hm i think this text should be part of the email. i think. Do you think we need it twice?

Reply:
I think we should definitely add more context to the website as well.

Reply:
I think there should be a text to introduce what the application is all about. Preferably it should not be the same text as in the email. For now I used the proposed text by @michael-markl to finish this pr.